app/src/main/java/org/fossasia/openevent/utils/CircleTransform.java

```diff
@@ -24,7 +24,7 @@ public Bitmap transform(Bitmap source) {
             source.recycle();
         }
 
-        Bitmap bitmap = Bitmap.createBitmap(size, size, source.getConfig());
+        Bitmap bitmap = Bitmap.createBitmap(size, size, Bitmap.Config.ARGB_8888);
```

Comment:
@creativepsyco as commented by me and @mananwason the stack trace is as stated below.Let me know if you want more detailed one but the null pointer was due to the below-stated reason only.